### PR TITLE
feat: Premium tests 

### PR DESCRIPTION
**Pull request by KenzoBenzo**
In this pull request:
- The project dashboard is changed to now splitting premium from continuous tests, and adding a new badge to mark premium tests.
- Individual test pages get a bit of a makeover with more relevant information.
- Added for premium tests, there is now fields for priority and comments. Comments is markdown, parsed.
- A custom code theme was added.

### CODE REVIEW

File: src/pages/[teamName]/[projectName]/[testId].tsx [hunk 1]
```diff
@@ -229,34 +236,44 @@ const TestPage = withError<GET_SERVER_SIDE_PROPS_ERROR, ITestProps>(
...
-                      : status === "Failed"
-                      ? "red"
-                      : null
-                  }
-                >
-                  {location}@
-                  <Link
-                    href={`https://github.com/${teamName}/${projectName}/commit/${commitHash}`}
+                {failures.length} Test Failure
+                {failures.length === 1 ? null : "s"}
```
Comment (mikesol) [inline src/pages/[teamName]/[projectName]/[testId].tsx:240]:
```suggestion
                {failures.length === 1 ? "Test Failure" : "Test Failures"}
```

File: src/utils/testLog.ts [hunk 2]
```diff
@@ -12,11 +12,12 @@ const exchangeType = t.type({
   }),
   response: t.type({
     statusCode: t.Integer,
+    body: t.union([t.string, t.undefined]),
   }),
```
Comment (mikesol) [inline src/utils/testLog.ts:16]:
```suggestion
  response: t.intersection([t.type({
    statusCode: t.Integer,
  }), t.partial({ body: t.string })]),
```

File: src/utils/testLog.ts [hunk 3]
```diff
@@ -27,6 +28,8 @@ const v1 = t.type({
     t.type({
       success: t.boolean,
       error_message: t.string,
+      comment: t.union([t.string, t.undefined]),
+      priority: t.union([t.Integer, t.undefined]),
       exchange: t.array(exchangeType),
     })
```
Comment (mikesol) [inline src/utils/testLog.ts:34]:
```suggestion
    t.intersection([
    t.type({
      success: t.boolean,
      error_message: t.string,
      exchange: t.array(exchangeType),
    }),
    t.partial({
      comment: t.string,
      priority: t.Integer
    })])
```

File: src/pages/[teamName]/[projectName]/[testId].tsx (hunk 1 above)
Comment (KenzoBenzo) [reply src/pages/[teamName]/[projectName]/[testId].tsx:240]: @mikesol Are you not wanting to display the count of failures then?

**Pull request merged**